rpm/SPECS/contextBroker.spec

```diff
@@ -31,12 +31,12 @@ Release:   %{broker_release}
 License:   AGPLv3
 Group:     Applications/Engineering
 Vendor:     Telefónica I+D
-Packager:   Fermín Galán <fermin@tid.es>
+Packager:   Fermín Galán <fermin.galanmarquez@telefonica.com>
 URL:        http://catalogue.fiware.org/enablers/publishsubscribe-context-broker-orion-context-broker
 Source:     %{name}-%{broker_version}.tar.gz
 BuildRoot: /var/tmp/%{name}-buildroot
-Requires:  libstdc++, boost-thread, boost-filesystem, libmicrohttpd, libcurl, logrotate
-Buildrequires: gcc, cmake, gcc-c++, libmicrohttpd-devel, libcurl-devel, boost-devel
+Requires:  libstdc++, boost-thread, boost-filesystem, gnutls, libcurl, logrotate
+Buildrequires: gcc, cmake, gcc-c++, gnutls-devel, libcurl-devel, boost-devel
```

Comment:
Some testing done at CentOS.

```
[fermin@centollo fiware-orion]$ rpm -q libmicrohttpd
libmicrohttpd-0.9.33-4.el6.x86_64
[fermin@centollo fiware-orion]$ rpm -q --requires libmicrohttpd
/sbin/ldconfig  
/sbin/ldconfig  
info  
info  
libc.so.6()(64bit)  
libc.so.6(GLIBC_2.10)(64bit)  
libc.so.6(GLIBC_2.2.5)(64bit)  
libc.so.6(GLIBC_2.3.2)(64bit)  
libc.so.6(GLIBC_2.3.4)(64bit)  
libc.so.6(GLIBC_2.4)(64bit)  
libc.so.6(GLIBC_2.9)(64bit)  
libdl.so.2()(64bit)  
libgcrypt.so.11()(64bit)  
libgcrypt.so.11(GCRYPT_1.2)(64bit)  
libgnutls.so.26()(64bit)  
libgnutls.so.26(GNUTLS_1_4)(64bit)  
libgpg-error.so.0()(64bit)  
libmicrohttpd.so.10()(64bit)  
libpthread.so.0()(64bit)  
libpthread.so.0(GLIBC_2.12)(64bit)  
libpthread.so.0(GLIBC_2.2.5)(64bit)  
librt.so.1()(64bit)  
librt.so.1(GLIBC_2.2.5)(64bit)  
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rtld(GNU_HASH)  
rpmlib(PayloadIsXz) <= 5.2-1
```

A little obscure... as rpm tool doesn't correlates this information to know to which RPM each file belongs.

Maybe an "empirical test" done in a just installed CentOS 6.x in FIWARE Lab once this PR gets merged would be more fruitful.